devsetup/scripts/gen-edpm-baremetal-kustomize.sh

```diff
@@ -70,15 +70,12 @@ fi)
     - op: add
       path: /spec/roles/edpm-compute/nodeTemplate/ansibleSSHPrivateKeySecret
       value: dataplane-ansible-ssh-private-key-secret
-    - op: replace
+    - op: remove
```

Comment:
We could just remove these 4 vars from the samples in dataplane-operator as well. Then we wouldn't need to remove them here.

Reply:
I'll start a PR then

Reply:
https://github.com/openstack-k8s-operators/dataplane-operator/pull/331